rebasehelper/application.py

```diff
@@ -846,18 +841,22 @@ def run(self):
         if not self.conf.patch_only:
             if not self.conf.comparepkgs:
                 # Build packages
-                try:
-                    if self.conf.build_tasks is None:
-                        self.build_source_packages()
-                    self.run_package_checkers(self.results_dir, category='SRPM')
-                    self.build_binary_packages()
-                    if self.conf.builds_nowait and not self.conf.build_tasks:
-                        return
-                    self.run_package_checkers(self.results_dir, category='RPM')
-                # Print summary and return error
-                except RebaseHelperError as e:
-                    self.print_summary(e)
-                    raise
+                while True:
+                    try:
+                        if self.conf.build_tasks is None:
+                            self.build_source_packages()
+                        self.run_package_checkers(self.results_dir, category='SRPM')
+                        self.build_binary_packages()
+                        if self.conf.builds_nowait and not self.conf.build_tasks:
+                            return
+                        self.run_package_checkers(self.results_dir, category='RPM')
+                        break
```

Comment:
Just a nitpick, but I think this `break` fits more into a `else` block of the `try ... except` statement.